PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+#### What does this PR do?
+#### Where should the reviewer start?
+#### How should this be manually tested?
+#### Any background context you want to provide?
+#### Screenshots (if appropriate)
+#### Questions:
+- Have you connected this PR to the issue it fixes?
```

Comment:
I'd use "resolves" instead of "fixes". 
Not all PRs are for defect.

Reply:
+1

Reply:
Yep, good call again. Changed to "resolves".